src/App.scss

```
@@ -575,6 +575,14 @@
   width: 16px;
 }
 
+.uni-FiltersButton-Count {
+  background: rgba(255, 255, 255, 0.32);
+  border-radius: 10px;
```

Good trick to keep this fully round even if we change the size of the badge.

```suggestion
  border-radius: 9999px;
```